NamespaceFixer/NamespaceBuilder/XamlNamespaceBuilderService.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.IO;
+using System.IO.Extensions;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Xml;
+
+namespace NamespaceFixer.NamespaceBuilder
+{
+    internal class XamlNamespaceBuilderService : INamespaceBuilder
+    {
+        public XamlNamespaceBuilderService(INamespaceAdjusterOptions options)
+        {
+            _options = options;
+        }
+
+        protected Match FindNamespaceMatch(string fileContent)
+        {
+            return Regex.Match(fileContent, @"x:Class=""([a-zA-Z0-9_\.]+)\.[a-zA-Z0-9_]+""");
```

Comment:
This finds the class name not the namespace. The project namespace is under `xmlns:local`. Both need to be fixed.

Reply:
Updated:
```csharp
protected Match FindClassNameMatch(string fileContent)
{
    return Regex.Match(fileContent, @"x:Class=""([a-zA-Z0-9_\.]+)\.[a-zA-Z0-9_]+""");
}

protected Match FindNamespaceMatch(string fileContent)
{
    return Regex.Match(fileContent, @"xmlns:local=""clr-namespace:([a-zA-Z0-9_\.]+)""");
}

// (...)

public bool UpdateFile(ref string fileContent, string desiredNamespace)
{
    if (string.IsNullOrEmpty(desiredNamespace)) return false;

    return 
        UpdateClassName(ref fileContent, desiredNamespace) ||
        UpdateNamespace(ref fileContent, desiredNamespace);
}
```
[commit link](https://github.com/pauer24/VsNamespaceFixer/pull/35/commits/0e1e070bd57b0026d9d6bdbc78d26468fd576e13)